libsodium-sys/build.rs

```diff
@@ -427,4 +427,7 @@ fn build_libsodium() {
 
     println!("cargo:rustc-link-lib=static=sodium");
     println!("cargo:rustc-link-search=native={}/lib", install_dir);
+    // Accessible externally via `DEP_SODIUM_INCLUDE` env.
```

Comment:
Are you sure this is enough to set env DEP_SODUIM_INCLUDE and DEP_ZMQ_LI**D**?

Reply:
For my use case this is enough.

But if we wanted to be rigorous we could add `DEP_SODIUM_ROOT` which would hold `OUT_DIR` and maybe `DEP_SODIUM_PKG_CONFIG` which would point to the pkg-config file.

Reply:
I am not sure you can set env var using println! in build.rs

Reply:
I think its quite old actually. I'll try to find the actual PR where this feature was implemented.

Reply:
I see, thanks.

Reply:
Couldn't find the PR, but based on the git blame of `libz-sys`, I know its a least 4 years old. But its quite a underground feature.